### PR TITLE
Java11 jogl

### PR DESCRIPTION
**Pull request by markpet49**
Numerous Javadoc, Java 11, and JOGL upgrade fixes.

### CODE REVIEW

**Comment by PJHogan** [comment]
Rock On!

File: lib-external/jogl-gluegen/build-gluegen.sh [hunk 1]
```diff
@@ -0,0 +1,5 @@
+set -x
+
+cd gluegen/make
+ant -Dtarget.sourcelevel=1.8 -Dtarget.targetlevel=1.8 -Dtarget.rt.jar=/home/mpeterson/d/jopengl/dummy.jar
```
Comment (Beak-man) [inline lib-external/jogl-gluegen/build-gluegen.sh]: Isn't there a problem with this referring to a local folder in your workstation?

File: lib-external/jogl-gluegen/build-jogl.sh [hunk 2]
```diff
@@ -0,0 +1,11 @@
+set -x
+
+cp patch/MacOSXJAWTWindow.java jogl/src/nativewindow/classes/jogamp/nativewindow/jawt/macosx
+cp patch/MacOSXCGLContext.java jogl/src/jogl/classes/jogamp/opengl/macosx/cgl
+cp patch/GLCanvas.java jogl/src/jogl/classes/com/jogamp/opengl/awt
+
+cd jogl/make
+ant -Dtarget.sourcelevel=1.8 -Dtarget.targetlevel=1.8 -Dtarget.rt.jar=/home/mpeterson/d/jopengl/dummy.jar
```
Comment (Beak-man) [inline lib-external/jogl-gluegen/build-jogl.sh]: Another reference to a local folder.

File: lib-external/jogl-gluegen/clean-gluegen.sh [hunk 3]
```diff
@@ -0,0 +1,5 @@
+set -x
+
+cd gluegen/make
+ant clean -Dtarget.sourcelevel=1.8 -Dtarget.targetlevel=1.8 -Dtarget.rt.jar=/home/mpeterson/d/jopengl/dummy.jar
```
Comment (Beak-man) [inline lib-external/jogl-gluegen/clean-gluegen.sh]: Reference to local folder.

File: lib-external/jogl-gluegen/clean-jogl.sh [hunk 4]
```diff
@@ -0,0 +1,5 @@
+set -x
+
+cd jogl/make
+ant clean -Dtarget.sourcelevel=1.8 -Dtarget.targetlevel=1.8 -Dtarget.rt.jar=/home/mpeterson/d/jopengl/dummy.jar
```
Comment (Beak-man) [inline lib-external/jogl-gluegen/clean-jogl.sh]: Reference to local folder.

File: lib-external/jogl-gluegen/linux-copy-jogl.sh [hunk 5]
```diff
@@ -0,0 +1,9 @@
+set -x 
+
+cp gluegen/build/gluegen-rt-natives-linux-amd64.jar ~/d/nasa/WorldWindJava
+cp gluegen/build/gluegen-rt.jar ~/d/nasa/WorldWindJava
```
Comment (Beak-man) [inline lib-external/jogl-gluegen/linux-copy-jogl.sh]: Reference to local folder?

File: src/gov/nasa/worldwind/formats/csv/CSVReader.java [hunk 6]
```diff
@@ -51,9 +51,9 @@ public int getNumPoints()
     }
 
     /**
-     * @param path
+     * @param path File spec to read from.
```
Comment (Beak-man) [inline src/gov/nasa/worldwind/formats/csv/CSVReader.java]: Excellent commenting

**Comment by markpet49** [comment]
@Beak-man Hi Miguel, I updated the PR to remove the local folder references.

File: src/gov/nasa/worldwind/formats/georss/GeoRSSParser.java [hunk 7]
```diff
@@ -319,7 +319,7 @@ private static Renderable makePolygonShape(Node node, Iterable<Node> attrs)
             return null;
         }
 
-        ArrayList<LatLon> positions = new ArrayList<LatLon>();
+        ArrayList<LatLon> positions = new ArrayList<>();
```
Comment (Beak-man) [inline src/gov/nasa/worldwind/formats/georss/GeoRSSParser.java]: I've been getting the following message regarding this line:
`Error:(322, 53) java: diamond operator is not supported in -source 6
  (use -source 7 or higher to enable diamond operator)`
Did a fresh install of oraclejdk11 and a fresh clone, to no avail.

**Pull request merged**